blueprints/common/partials/imagebuilder_component.hbs

```diff
@@ -0,0 +1,46 @@
+{{!--
+  Creates an Image Builder Component
+https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-imagebuilder-component.html
+--}}
+
+"imagebuilder{{sanitize this.Name ''}}":
+{
+  "Type" : "AWS::ImageBuilder::Component",
+
+  "Properties" : {
+      {{#if this.ChangeDescription}}
+        "ChangeDescription" : "{{this.ChangeDescription}}",
+      {{/if}}
+
+      {{#if this.Data}}
+        "Data" : "{{this.Data}}",
+      {{/if}}
+
+      {{#if this.Description}}
+        "Description" : "{{this.Description}}",
+      {{/if}}
+
+      {{#if this.KmsKeyId}}
+        "KmsKeyId" : "{{this.KmsKeyId}}",
+      {{/if}}
+
+        "Name" : "{{this.Name}}",
+
+        "Platform" : "{{this.Platform}}",
+
+      {{#if this.SupportedOsVersions}}
+        "SupportedOsVersions" :
+        {{#each this.SupportedOsVersions}}
+          "{{this}}"{{#unless @last}},{{/unless}}
```

Comment:
Fun fact: after writing {{#unless @last}},{{/unless}} too many times, I created a shortcut for it named {{comma}} that you can use instead.